Doctrine/Orm/Filter/SearchFilterExact.php

```diff
@@ -34,4 +37,20 @@ protected function normalizeValues(array $values, string $property): ?array
 
         return array_values($values);
     }
+
+    protected function addWhereByStrategy(string $strategy, QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $alias, string $field, $values, bool $caseSensitive)
```

Comment:
Is it not a better idea to lint this method as follows?

`protected function addWhereByStrategy(
                string $strategy, 
               QueryBuilder $queryBuilder, 
               QueryNameGeneratorInterface $queryNameGenerator, 
               string $alias,  
               string $field, 
               $values, bool $caseSensitive)`

Reply:
:+1: